instance.go

```diff
@@ -766,6 +772,91 @@ func (opts BulkDropOptions) Concurrency() int {
 	return opts.MaxConcurrency
 }
 
+// DropViewsInSchema drops all tables in a schema. If opts.OnlyIfEmpty==true,
+// returns an error if any of the tables have any rows.
+func (instance *Instance) DropViewsInSchema(schema string, opts BulkDropOptions) error {
+	ctx := context.Background()
+	db, err := instance.CachedConnectionPool(schema, opts.params())
+	if err != nil {
+		return err
+	}
+
+	// Obtain view names
+	var views []string
+	if opts.Schema != nil {
+		for _, v := range opts.Schema.Views {
+			views = append(views, v.Name)
+		}
+	} else {
+		views, err = getViewNames(ctx, db, schema)
+		if err != nil {
+			return err
+		}
+	}
+	if len(views) == 0 {
+		return nil
+	}
+
+	// If requested, confirm tables are empty
+	if opts.OnlyIfEmpty {
+		names := make([]string, 0, len(views))
+		for _, tableName := range views {
+			names = append(names, tableName)
+		}
+		if err := confirmTablesEmpty(db, names); err != nil {
+			return err
+		}
+	}
+
+	// If buffer pool is over 32GB and flavor doesn't have optimized DROP TABLE,
+	// reduce drop concurrency to 1 to reduce risk of stalls
+	concurrency := opts.Concurrency()
+	if instance.bufferPoolSize >= (32*1024*1024*1024) && !instance.flavor.MySQLishMinVersion(8, 0, 23) {
```

Comment:
It'd be more readable IMHO to extract `32*1024*1024*1024` into a descriptive constant.

Reply:
Done